projects/authentication/docs/user-access-token.md

```diff
@@ -303,7 +304,7 @@ The following table gives an overview of all the required/optional parameters:
 | `scope`                 | A space-delimited string with one or more of the following values: `openid`, `email`, `offline_access`                                                                                               | Required if you want to fetch **user info** or use **refresh tokens** | Determines which user info will be shared with your application, and if you want a refresh token or not. Suggested to always be set to `openid email offline_access` for best results. See [scopes](#scopes) for more info.                                      |
 | `code_challenge`        | String generated from a cryptographically-random code verifier.                                                                                                                                      | Required if using **PKCE**                                            | See the [PKCE example](#example-1) for more info.                                                                                                                                                                                                                |
 | `code_challenge_method` | `S256`                                                                                                                                                                                               | Required if using **PKCE**                                            | **Must** be set to `S256`. See the [PKCE example](#example-1) for more info.                                                                                                                                                                                     |
-| `prompt`                | `none` (default) or `login`                                                                                                                                                                          | Optional                                                              | If set to `login`, the user will always be asked to login even if they still have an active session on the authorization server. Useful to force the user to login again, which also makes it possible to switch to another account as before.                   |
+| `prompt`                | `login` or omitted (default)                                                                                                                                                                          | Optional                                                              | If set to `login`, the user will always be asked to login even if they still have an active session on the authorization server.                   |
```

Comment:
Dit is veel duidelijker, het leek erop dat de waarde 'none' moest worden ingevoerd (terwijl het niet zo was).